proto/server/configEvtHandler.go

```diff
@@ -210,6 +215,28 @@ func handleNetworkSlicePost(configMsg *configmodels.ConfigMessage, subsUpdateCha
 	if errPost != nil {
 		logger.DbLog.Warnln(errPost)
 	}
+	if factory.WebUIConfig.Configuration.SendPebbleNotifications {
+		sendPebbleNotification("canonical.com/webconsole/networkslice/create")
+	}
+	rwLock.Unlock()
+}
+
+func handleNetworkSliceDelete(configMsg *configmodels.ConfigMessage, subsUpdateChan chan *Update5GSubscriberMsg) {
+	rwLock.Lock()
+	if factory.WebUIConfig.Configuration.Mode5G {
+		var config5gMsg Update5GSubscriberMsg
+		config5gMsg.Msg = configMsg
+		config5gMsg.PrevSlice = getSliceByName(configMsg.SliceName)
+		subsUpdateChan <- &config5gMsg
+	}
+	filter := bson.M{"SliceName": configMsg.SliceName}
+	errDelOne := dbadapter.CommonDBClient.RestfulAPIDeleteOne(sliceDataColl, filter)
+	if errDelOne != nil {
+		logger.DbLog.Warnln(errDelOne)
+	}
+	if factory.WebUIConfig.Configuration.SendPebbleNotifications {
+		sendPebbleNotification("canonical.com/webconsole/networkslice/delete")
```

Comment:
Should not this be a configurable parameter instead of a hard-coded value?

Reply:
At this point I don't see a need for parameterizing the key, because our charmed operators will be the only user of this feature. In the future, if it becomes troublesome, we could think about building this string out of configurable elements.